tests/test_qube_ext_tvl.py

```
@@ -0,0 +1,721 @@
+        result = a.tvl_gt(4.0)
+        self.assertEqual(result, Boolean(True))
+
+        result = a.tvl_le(6.0)
+        self.assertEqual(result, Boolean(True))
+
+        result = a.tvl_ge(4.0)
+        self.assertEqual(result, Boolean(True))
+
+        # Test with masked self and non-Qube arg
+        # Note: When builtins is enabled, masked comparisons might return bool
+        Qube.prefer_builtins(False)
```

Ruff (0.14.7)

Boolean positional value in function call

(FBT003)